usaspending_api/references/models.py

```diff
@@ -63,6 +63,15 @@ class Agency(models.Model):
     subtier_agency = models.ForeignKey('SubtierAgency', models.DO_NOTHING, null=True)
     office_agency = models.ForeignKey('OfficeAgency', models.DO_NOTHING, null=True)
 
+    @staticmethod
+    def get_default_fields():
+        return [
+            "id",
```

Comment:
I don't think it's needed, as the cgac/fpds/subtier codes should be enough to uniquely identify them at that point; and the agencies most people will be searching on is the combination of the 3